rfc/kubernetes-ui-enhancements/kubernetes-ui-enhancements.md

```diff
@@ -0,0 +1,260 @@
+
+### Daemon Sets / Stateful Sets
+#### Current Functionality
+
+Daemon Sets / Stateful Sets are currently rendered in Spinnaker’s infrastructure view. They leverage a similar UI to Replica Sets and successfully convey all key object attributes. Given that the Daemon Set / Stateful Set UI has near parity with other tooling, I recommended leaving this functionality unchanged for this project.
+
+### Namespaces
+#### Current Functionality
+
+There is no dedicated rendering of Namespace metadata or associated Resource Quotas in Spinnaker’s UI. Namespace information is only provided as a string identifying where a deployed object resides. While it would be possible to add functionality for exposing associated Resource Quota objects, this feature would be primarily of use to cluster administrators. Refactoring the Spinnaker UI to surface Namespace resource allocation would be of limited use to application teams.
+
+## Terminology Alignment Proposal
+
+### Scope
+
+The changes proposed here are scoped to Spinnaker terminology that is generic and ambiguous when applied to the Kubernetes provider. This is especially problematic in Spinnaker’s infrastructure view where overloaded terms, such as “Clusters”, are misapplied.
+Reworking the terminology for pipeline stages is out of scope for phase 1 of this effort. Existing stages are already specific to the Kubernetes provider and render tooltips to guide users on their functionality.
+
+### Prior Art
+
+Special thanks to [demobox](https://github.com/demobox) and [maggieneterval](https://github.com/maggieneterval) whose prior research into Spinnaker's Kubernetes terminology was helpful in drafting these terminology recommendations.
+
+### Terminology Changes
+
+#### Infrastructure view
+
+Cluster → Workloads
+
+Spinnaker uses the term “cluster” in the infrastructure view as an all encompassing term for the metadata, instances, and environments that comprise an application. This term is incongruent with the Kubernetes definition of a cluster. The term “workload” aligns Spinnaker with other tools and accurately refers to the set of Kubernetes application resources.
+
+Load Balancers
+
+The load balancer tab of Spinnaker’s infrastructure view is used to display deployed Service and Ingress objects. While updating this to “Services & Ingresses” would bring Spinnaker into alignment with other tools, it would require further tweaking if the category is expanded to include future object types. Unlike Spinnaker’s overloaded usage of the term “cluster”, the load balancer label is not applied in direct conflict with a pre-existing Kubernetes specific term.
```

Comment:
I think you're proposing to leave Load Balancers alone ... which is fine with me, but not totally clear.  It's also in a section called `Terminology Changes`, so if it's not a change, it's a little confusing.